pkg/docker/config/config.go

```diff
@@ -353,6 +494,31 @@ func modifyJSON(sys *types.SystemContext, editor func(auths *dockerConfigFile) (
 	return nil
 }
 
+// errCredNotConfigured indicates the credential service for the registry was not configured
+var errCredNotConfigured = errors.New("Credential service configuration not found")
+
+// getRegistryCredConfigs returns the credHelper configed for the registry
+// creds_store will not be used if cred_helper configured for specified registries
+func getRegistryCredConfigs(sys *types.SystemContext, registry string) ([]string, error) {
```

Comment:
Naming: `config` does not really say anything specific; saying `CredentialHelper[s]` in the name would be useful.